exporter/fairshare_test.go

```diff
@@ -0,0 +1,113 @@
+package exporter
+
+import (
+	"testing"
+)
+
+func TestParseFairShare(t *testing.T) {
```

Comment:
Let's use the `assert` and `require` pkg's here to be more expressive and inline with the rest of the pkg


```suggestion
func TestParseFairShare(t *testing.T) {
    require := require.New(t)
    assert := assert.New(t)
```